DOCSP 35172 Stage Diagram Task Pages TOC (#77)
mongodb/docs-relational-migrator: source/diagrams/manage-diagrams.txt

.. _manage_diagrams:

===============
Manage Diagrams
===============

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 1
   :class: singlecol

> The following pages outline the details and specifics of managing
> diagrams use with Relational Migrator.

Reviewer: ```suggestion
The following pages outline the details and specifics of how to manage diagrams
in Relational Migrator.
```

Author: Done!
